core-web/libs/sdk/angular/src/lib/utils/index.ts

```diff
@@ -0,0 +1,69 @@
+import { DotCMSContainer, DotCMSPageAssetContainer } from '../models';
+
+const endClassMap: Record<number, string> = {
+    1: 'col-end-1',
+    2: 'col-end-2',
+    3: 'col-end-3',
+    4: 'col-end-4',
+    5: 'col-end-5',
+    6: 'col-end-6',
+    7: 'col-end-7',
+    8: 'col-end-8',
+    9: 'col-end-9',
+    10: 'col-end-10',
+    11: 'col-end-11',
+    12: 'col-end-12',
+    13: 'col-end-13'
+};
+
+const startClassMap: Record<number, string> = {
+    1: 'col-start-1',
+    2: 'col-start-2',
+    3: 'col-start-3',
+    4: 'col-start-4',
+    5: 'col-start-5',
+    6: 'col-start-6',
+    7: 'col-start-7',
+    8: 'col-start-8',
+    9: 'col-start-9',
+    10: 'col-start-10',
+    11: 'col-start-11',
+    12: 'col-start-12'
+};
+
+export const getContainersData = (
+    containers: DotCMSPageAssetContainer,
+    containerRef: DotCMSContainer
+) => {
+    const { identifier, uuid } = containerRef;
+
+    const { containerStructures, container } = containers[identifier];
+
+    // Get the variant id
+    const { variantId } = container?.parentPermissionable || {};
+
+    // Get accepts types of content types for this container
+    const acceptTypes: string = containerStructures
+        .map((structure) => structure.contentTypeVar)
+        .join(',');
+
+    // Get the contentlets for "this" container
+    const contentlets = containers[identifier].contentlets[`uuid-${uuid}`];
+
+    return {
+        ...containers[identifier].container,
+        acceptTypes,
+        contentlets,
+        variantId
+    };
+};
+
+export const getPositionStyleClasses = (start: number, end: number) => {
+    const startClass = startClassMap[start];
+    const endClass = endClassMap[end];
+
+    return {
+        startClass,
+        endClass
+    };
+};
```

Comment:
We should consider centralize this, could be useful for future implementations or even clients


cc: @fmontes

Reply:
@kevindaviladev was mentioning having a library for this thing because SQ was crying about it, but if we think we should do it for the SDK, worth to consider.

Reply:
I think it will be necessary to create this utility library, so as not to run into many changes in the SDKs that we will create.

With the path strategy in tsconfig, linking in dev mode should be easier

But I would suggest creating a separate ticket for this new lib and working the migration from there

Reply:
I agree, the migration of this code should be another ticket.